src/main/webapp/scripts/session-script.js

```diff
@@ -131,6 +244,43 @@ function changeControllerTo(event, controller) {
   }
 }
 
+/**
+ * function updateController() checks to see if the current user should
+ * be the controller of their party, changing session screen privilege
+ * and updating user interface.
+ * @param {string} controller name of the controller of the session
+ */
+function updateController(controller) {
+  const /** HTMLElement */ sessionInfoAttendeesDiv =
+      document.getElementById('session-info-attendees');
+  const /** NodeListOf<HTMLSpanElement> */ controllerToggleList = 
+      sessionInfoAttendeesDiv.querySelectorAll('span');
+  if (urlParameters.get(URL_PARAM_KEY.SCREEN_NAME) === controller) {
+    noVNCClient.setViewOnly(false);
+  }
+  controllerToggleList.forEach(individualSpanElement => {
+    individualSpanElement.style.backgroundColor = '#fff';
+  });
+  sessionInfoAttendeesDiv.querySelector(`#${controller}`)
+          .parentElement.querySelector('span').style.
+              backgroundColor = '#fd5d00';
```

Comment:
What are these doing? Add comments of separate into another appropriately named function